Prop that displays a search bar to filter facets (#411)
vtex-apps/search-result: docs/README.md

| `blockClass`        | `string`       | Unique block ID to be used in [CSS customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property)                                                                                    | `undefined`              |
| `trackingId` | `string` | ID to be used in Google Analytics to track store metrics based on the Search Result block. |  `Search result` | 
| `mobileLayout`      | `Object` | Controls how the search results page will be displayed to users when using the mobile layout. Possible props and their respective values can be found below.                                                                                                                | `undefined`              |

> | `showFacetSearch` | `number` | Minimum number of options that must exist in a filter for the search bar to be displayed. If it is `0`, the search bar will always be displayed. |  `undefined` | 

Reviewer: The name of this prop suggests that it's a boolean value.

Maybe we can change to something like `searchFacetMinimum`? Open to other names

Author: what do you think about `minimumSearchThreshold` or something like this?

Reviewer: Like it!

Reviewer: Or `thresholdForFacetSearch`

I think the word facet is important here

> | `thresholdForFacetSearch` | `number` | Minimum number of options that must exist in a filter for the search bar to be displayed. If it is `0`, the search bar will always be displayed. |  `undefined` | 

Reviewer 2: ```suggestion
| `thresholdForFacetSearch` | `number` | Minimum number of filters that must be displayed on the interface for a search bar to be displayed. If you declare `0`, the search bar will always be displayed. |  `undefined` | 
```
